client/app/bundles/course/assessment/pages/AssessmentStatistics/LiveFeedbackHistory/LiveFeedbackHistoryPage.tsx

```diff
@@ -0,0 +1,95 @@
+import { FC, useState } from 'react';
+import { defineMessages } from 'react-intl';
+import { Slider, Typography } from '@mui/material';
+
+import Accordion from 'lib/components/core/layouts/Accordion';
+import { useAppSelector } from 'lib/hooks/store';
+import useTranslation from 'lib/hooks/useTranslation';
+import { formatLongDateTime } from 'lib/moment';
+
+import {
+  getLiveFeedbackHistory,
+  getLiveFeedbadkQuestionInfo,
+} from '../selectors';
+
+import LiveFeedbackDetails from './LiveFeedbackDetails';
+
+const translations = defineMessages({
+  questionTitle: {
+    id: 'course.assessment.liveFeedback.questionTitle',
+    defaultMessage: 'Question {index}',
+  },
+});
+
+interface Props {
+  questionNumber: number;
+}
+
+const LiveFeedbackHistoryPage: FC<Props> = (props) => {
+  const { t } = useTranslation();
+  const { questionNumber } = props;
+  const allLiveFeedbackHistory = useAppSelector(getLiveFeedbackHistory).filter(
```

Comment:
please split this into 2 lines
```
const allLiveFeedbackHistory = useAppSelector(getLiveFeedbackHistory)
const nonEmptyLiveFeedbackHistory = allLiveFeedbackHistory.filter((liveFeedbackHistory) => {
      // Remove live feedbacks that have no comments
      return liveFeedbackHistory.files.some((file) => file.comments.length > 0);
    })
```

the way it's done now is quite dirty, and we have never used `useAppSelector` this way, creating inconsistencies

Reply:
done